tests/legacyjit_x86_testenv.cmd

```diff
@@ -0,0 +1,11 @@
+@REM -------------------------------------------------------------------------
+@REM 
+@REM  This script provides x86 Legacy JIT test environment settings
+@REM
+@REM -------------------------------------------------------------------------
+
+set COMPLUS_AltJit=*
+set COMPLUS_AltJitNgen=*
+set COMPLUS_AltJitName=legacyjit.dll
+set COMPLUS_NoGuiOnAssert=1
+set COMPLUS_AltJitAssertOnNYI=1
```

Comment:
GitHub tells me there is no newline at the end of this file. Will that be a problem?